### ISSUE TITLE
[bug] NoClassDefFoundError in RHEL 6.8

### ISSUE THREAD

**Issue by nerophon** [elastic/support-diagnostics#253]
The ES diagnostic tool fails at runtime with the following:
```
Exception in thread "main" java.lang.NoClassDefFoundError: Could not initialize class oshi.jna.platform.liux.Libc 
```

Context:
`support-diagnostics-6.4-2.4`
`RHEL 6.8`
`java version 1.8.0_131-b11`

I found a possible cause here:
https://github.com/oshi/oshi/issues/338
TL;DR: JNA now appears to require glibc version 2.14, but CentOS 6 has version 2.12. However an attempt was made to run the diagnostic in RHEL 7 with a later version of glibc, and it was reported to fail with the same error, so we may need to look further for the actual cause.

I've raised this here under the assumption that we want this tool to work with all Elastic-supported operating systems and versions.

**Comment by nemonster** [comment]
Addressed in 7.0 via removal of library that generated error.
